.teamcity/src/main/kotlin/model/CIBuildModel.kt

```diff
@@ -19,19 +19,19 @@ import configurations.TestPerformanceTest
 import projects.DEFAULT_FUNCTIONAL_TEST_BUCKET_SIZE
 import projects.DEFAULT_LINUX_FUNCTIONAL_TEST_BUCKET_SIZE
 
-enum class StageNames(override val stageName: String, override val description: String) : StageName {
-    QUICK_FEEDBACK_LINUX_ONLY("Quick Feedback - Linux Only", "Run checks and functional tests (embedded executer, Linux)"),
-    QUICK_FEEDBACK("Quick Feedback", "Run checks and functional tests (embedded executer, Windows)"),
-    READY_FOR_MERGE("Pull Request Feedback", "Run vairous functional tests against distribution") {
+enum class StageNames(override val stageName: String, override val description: String, override val uuid: String) : StageName {
+    QUICK_FEEDBACK_LINUX_ONLY("Quick Feedback - Linux Only", "Run checks and functional tests (embedded executer, Linux)", "QuickFeedbackLinuxOnly"),
+    QUICK_FEEDBACK("Quick Feedback", "Run checks and functional tests (embedded executer, Windows)", "QuickFeedback"),
+    READY_FOR_MERGE("Pull Request Feedback", "Run vairous functional tests against distribution", "PullRequestFeedback") {
```

Comment:
```suggestion
    READY_FOR_MERGE("Pull Request Feedback", "Run various functional tests", "PullRequestFeedback") {
```

typo + removed redundant information.